src/cast_core/src/Loader.cpp

```diff
@@ -76,6 +76,12 @@ struct Loader::PhaseConfig {
             }
         };
 
+        if (context["Threads"] && context["Threads"].to<IntegerSpec>() > totalThreads) {
```

Comment:
This should be on line 108. 

(Otherwise you will get a series of InvalidConfigurationException("Phase Config 'Threads' parameter must be less than  or equal to Actor Config 'Threads' in Loader actor")); followed by InvalidConfigurationException("Phase Config 'Threads' parameter is not supported if  MultipleThreadsPerCollection' is true.")); when multipleThreadsPerCollection is set)

Reply:
Ah, good catch. Thanks. Moving